src-tauri/src/configs/config_core.rs

```diff
@@ -143,10 +145,9 @@ impl ConfigCore {
         let mut config = Self::current().write().await;
         config.load_app_handle(app_handle.clone()).await;
 
-        if config.content.node_type.ne(&NodeType::RemoteUntilLocal) {
-            config
-                ._get_content_mut()
-                .set_node_type(NodeType::RemoteUntilLocal);
+        if config.content.migration_id.eq(&0) {
```

Comment:
Existing configs won't have this field, so it won't equal 0, it won't exist, or it would be default 1.